test/gtest/uct/ib/test_rc.cc

```diff
@@ -7,7 +7,6 @@
 
 #include "test_rc.h"
 #include <uct/ib/rc/verbs/rc_verbs.h>
-#include <uct/ib/rc/accel/rc_mlx5.h>
```

Comment:
you can leave the `include`, pls check the way it is included here
https://github.com/openucx/ucx/blob/master/test/gtest/uct/ib/test_ib_event.cc#L17

Reply:
added rc/dc/rc_verbs numerical values for ep iface address size, as requested in comments to pin size, instead of reusing same structure, so include is not needed.

Reply:
i think better to use sizeof of the relevant structs (like in the original commit) rather than defining size to ep name mapping
wdyt?
cc @yosefe

Reply:
squashing commit as @yosefe approved, will commit on top if needed